### PR TITLE
Edge to edge implementation for starter

### PR DESCRIPTION
**Pull request by the-scrambler**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -68,7 +68,7 @@ android {
 
 dependencies {
 
-    implementation(platform("androidx.compose:compose-bom:2023.05.01"))
+    implementation(platform("androidx.compose:compose-bom:2023.06.01"))
```
Comment (android-dev-lxl) [inline app/build.gradle.kts:71]: Latest is 2023.08.00

File: app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt [hunk 2]
```diff
@@ -167,14 +191,23 @@ fun BusScheduleScreen(
         Row(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(dimensionResource(R.dimen.padding_medium)),
+                .padding(dimensionResource(R.dimen.padding_medium))
+                .padding(
+                    PaddingValues(
+                        top = contentPadding.calculateTopPadding(),
+                    )
+                ),
```
Comment (android-dev-lxl) [inline app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt:199]: This needs to be updated in the main branch as well. 

Could we simplify this code?
.padding(top = contentPadding.calculateTopPadding())

https://github.com/google-developer-training/basic-android-kotlin-compose-training-bus-schedule-app/pull/7#discussion_r1290843497

File: app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt [hunk 3]
```diff
@@ -167,14 +191,23 @@ fun BusScheduleScreen(
...
             horizontalArrangement = Arrangement.SpaceBetween
         ) {
             Text(stopNameText)
             Text(stringResource(R.string.arrival_time))
         }
         Divider()
         BusScheduleDetails(
+            modifier = Modifier.padding(vertical = 8.dp),
```
Comment (android-dev-lxl) [inline app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt:207]: Could we use `padding_medium` here.

**Review by android-dev-lxl — COMMENTED** [review]
Please remove unused val: 
`val seed = Color(0xFF6750A4)
`
Given the min version is 24 so we do not need drawble-V24 folder

BusScheduleViewModel - remove empty construct

`class BusScheduleViewModel(): ViewModel() {` -> `class BusScheduleViewModel: ViewModel() { `

**Review by johnshea — CHANGES_REQUESTED** [review]
API 33 details UI is different than API 33 details UI on branch `starter`.

Screenshot 2023-08-21 at 12 22 06 PM

**Review by johnshea — CHANGES_REQUESTED** [review]
There are issue with the UI being noticeably different between branch `starter` and `starter-e2e` that need to be corrected.
- Spacing between the column labels and divider bar (More obvious on portrait mode).
- On landscape e2e, the time is indented further and does not line up with column label.

Note - I added additional stops to confirm nav bar was translucent.

Screenshot 2023-08-29 at 1 16 10 PM

Screenshot 2023-08-29 at 1 14 34 PM




Can build.gradle.kts(app) java version be upgraded to 1.8 for `sourceCompatibility`,`targetCompatibility`, and `jvmTarget`?

**Pull request closed without merging**